File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.py
class Solution:
    def searchRange(self, nums: List[int], target: int) -> List[int]:
        l=[]
        for i in range(len(nums)):
            if(target==nums[i]):
                l.append(i)
        k=[]
        if(len(l)==0):
            return [-1,-1]
        k.append(l[0])
        k.append(l[len(l)-1])
        
        return k

        

File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.cpp
class Solution {
public:
    vector<int> searchRange(vector<int>& nums, int target) {
        vector<int>l;
        for (int i=0;i<nums.size();i++){
            if(target==nums[i]){
                l.push_back(i);
            }
        }
        if(l.size()==0){
            l.push_back(-1);
            l.push_back(-1);
            return l;
        }
        vector<int>og;
        og.push_back(l[0]);
        og.push_back(l.back());
        return og;
        
    }
};